iChef-WebUi/src/app/components/recipes/recipes.component.ts

```diff
@@ -4,25 +4,80 @@ import { Router } from '@angular/router';
 import { tap } from 'rxjs/operators';
 import { Recipe } from 'src/app/data-types/recipe';
 import { RecipesService } from 'src/app/services/recipes.service';
-import { BUTTON_STRINGS, PLACEHOLDERS_STRINGS } from '../../constants/texts';
+import { BUTTON_STRINGS, PLACEHOLDERS_STRINGS, TITLES } from '../../constants/texts';
+import { trigger, state, style, transition, animate } from '@angular/animations';
+import { RecipeIngredient } from 'src/app/data-types/ingredient';
+import { Utensil } from 'src/app/data-types/utensil';
+import { IngredientsService } from 'src/app/services/ingredients.service';
+import { UtensilsService } from 'src/app/services/utensils.service';
+import { Filter } from 'src/app/data-types/filter';
+import { FilterCriteria } from 'src/app/data-types/filter-criteria';
 
 @Component({
     selector: 'app-recipes',
     templateUrl: './recipes.component.html',
-    styleUrls: ['./recipes.component.scss']
+    styleUrls: ['./recipes.component.scss'],
+    animations: [
```

Comment:
Can't you put this animation in a separate file? https://stackoverflow.com/questions/43096429/angular-animations-as-a-separate-file-not-in-lined